src/pubsub/mod_pubsub_db_rdbms.erl

```diff
@@ -577,35 +918,30 @@ get_entity_items(Nidx, { LU, LS, _ }) ->
     ok.
 remove_items(Nidx, { LU, LS, _ }, ItemIds) ->
     lists:foreach(fun(ItemId) ->
-                          SQL = mod_pubsub_db_rdbms_sql:delete_item(Nidx, LU, LS, ItemId),
-                          {updated, _} = mongoose_rdbms:sql_query_t(SQL)
-                  end, ItemIds).
+        {updated, _} = execute_delete_item(LS, Nidx, LU, ItemId)
+    end, ItemIds).
 
 -spec remove_all_items(Nidx :: mod_pubsub:nodeIdx()) ->
     ok.
 remove_all_items(Nidx) ->
-    SQL = mod_pubsub_db_rdbms_sql:delete_all_items(Nidx),
-    {updated, _} = mongoose_rdbms:sql_query_t(SQL),
+    {updated, _} = execute_delete_all_items(Nidx),
     ok.
```

Comment:
I looked at the callers of these functions as they essentially duplicate `del_item` and `del_items` and I found that they are called one after another in `node_flat`. In `mod_pubsub_db_mnesia` they are doing separate things:

- del_item(s) remove `pubsub_item` records
- remove_(all)_items remove items from the `pubsub_state` record

However, in RDBMS they are doing the same thing, so it is done twice. One solution might be to make `del_item` and `del_items` do nothing in the DB, but this is counter-intuitive, so I think that we could keep it as it is for now and fix it in `node_flat` in a separate PR.